### ISSUE TITLE
MvxSpinner documentation

### ISSUE THREAD

**Issue by MarcBruins** [MvvmCross/MvvmCross#2631]
### Steps to reproduce :scroll:

There is no documentation on the MvxSpinner.

### Expected behavior :thinking:
Simple documentation to explain how it works

### Actual behavior :bug:
Tell us what happens instead

### Configuration :wrench:

**Version:** 5.x

**Platform:** 
- [ ] :iphone: iOS
- [x] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by nickrandolph** [comment]
@MarcBruins do you feel this is something you could assist with? If so, please feel free to submit a PR with your contribution.

**Comment by Cheesebaron** [comment]
@MarcBruins if you start looking at one of these pieces of doc, can you please just write in this issue so I don't start on the same thing? Thanks!

**Comment by Divyesh1** [comment]
Yes, in Xamarin.iOS there are still missing controls like InputTextLayout, floating action button, checkboxes, radio buttons. @Cheesebaron

**Comment by Cheesebaron** [comment]
@Divyesh1 no. These controls are not native to iOS.

**Comment by Cheesebaron** [comment]
I will be looking at `MvxSpinner` next
